client/app/css/less/common/global.less

```diff
@@ -15,6 +15,16 @@ input:focus,
     -moz-outline: none;
 }
 
+.wizard-title {
+    line-height: rem-calc(22.4);
+    margin-top: 0;
+    margin-bottom: 0;
+    color: @oui-color-zodiac;
+    font-size: rem-calc(21);
+    font-weight: 600;
+    letter-spacing: 0.01em;
```

Comment:
`.01em` :candy:

Reply:
Done. It was a copypaste from ovh-ui-kit to replace &:extend(.modal__title).